Python/sysmodule.c

```diff
@@ -2641,6 +2641,51 @@ sys__baserepl_impl(PyObject *module)
     Py_RETURN_NONE;
 }
 
+/*[clinic input]
+sys._clear_type_descriptors
+
+    type: object
```

Comment:
You can use Argument Clinic for type checking. Although it does not make much difference.
```suggestion
    type: object(subclass_of="&PyType_Type")
```

Reply:
Thanks, done.